src/commands/deploy/title-escrow.ts

```diff
@@ -0,0 +1,71 @@
+import { Argv } from "yargs";
+import { error, success, info } from "signale";
+import { getLogger } from "../../logger";
+import { deployTitleEscrow } from "../../implementations/deploy/title-escrow";
+import { DeployTitleEscrowCommand } from "./deploy.types";
+
+const { trace } = getLogger("deploy:title-escrow");
+
+export const command =
+  "title-escrow <token-registry-address> <beneficiary> <holder> <titleEscrowFactoryAddress> [options]";
+
+export const describe = "Deploys a title escrow on the blockchain";
+
+export const builder = (yargs: Argv): Argv =>
```

Comment:
some inconsistency:
- beneficiary address is not suffixed by address unlike token registry ( I would get rid of all address suffix)
- `titleEscrowFactoryAddress` vs `token-registry-address`

Reply:
fixed